src/main/kotlin/racingcar/domain/Commentator.kt

```diff
@@ -0,0 +1,28 @@
+package racingcar.domain
+
+import racingcar.res.GuideMessage.ANNOUNCE_THE_WINNERS
+import racingcar.res.GuideMessage.SHOW_EXECUTION_RESULT
+import racingcar.utils.getRandomNumber
+
+class Commentator(private val cars: List<Car>, private val numberOfMoves: Int) {
+  fun commentRace() {
+    println()
+    println(SHOW_EXECUTION_RESULT)
+
+    repeat(numberOfMoves) {
+      for (car in cars) {
+        car.moveOrNot(getRandomNumber())
+        println("${car.carName} : ${car.record}")
+      }
+      println()
+    }
+  }
+
+  fun theWinnerIs() {
+    val maxRecord = cars.map{ it.record }.maxOf { it.length }
+    val nameOfWinners = cars.filter { it.record.length == maxRecord }.map { it.carName }
+
+    print(ANNOUNCE_THE_WINNERS)
+    println(nameOfWinners.joinToString(", "))
```

Comment:
```suggestion
    println(nameOfWinners.joinToString())
```
`joinToString`의 기본 separator가 아래와 같기 때문에 기본형으로 사용해도 될 것 같습니다.
```
public fun <T> Iterable<T>.joinToString(separator: CharSequence = ", ", prefix: CharSequence = "", postfix: CharSequence = "", limit: Int = -1, truncated: CharSequence = "...", transform: ((T) -> CharSequence)? = null): String {
    return joinTo(StringBuilder(), separator, prefix, postfix, limit, truncated, transform).toString()
}
```